taxonomies/ingredients.txt

```diff
@@ -53714,12 +53903,26 @@ zh:羽扇豆属, 羽扇豆, 鲁冰花
 # zh-sg:羽扇豆
 # zh-tw:羽扇豆
 allergens:en: en:lupin
-wikidata:en:Q156811
+#plant:wikidata:en:Q156811
+wikidata:en:Q13582643
 wikipedia:en:https://en.wikipedia.org/wiki/Lupin_bean
-# ingredient/fr:lupin has 390 products @2019-04-11
-# usage:fr:lupins (lupine, sel de cuisine, antioxydants:acide citrique et acide ascorbique, conservateur:E202)
-
-
+ciqual_food_code:en:20534
+ciqual_food_name:en:lupin grain, raw
+ciqual_food_name:fr:Lupin, graine crue
+# 957 in 12 @2021-10-01
+
+#processing:en: en:flour
```

Comment:
I think we should keep the "(something) flour" entries as separate ingredients.

Reply:
we can, but why? If it is a separate ciqual I do keep the entry

Reply:
It makes it possible to have "flour" as a parent, and to find foods that contain "(something) flour" (and not all the foods containing "(something)".

For the processings, I think we should include only processings that don't transform the ingredient into something radically different. e.g. "cut something", "dried something" etc. "something juice" is probably fine too. But for "something flour" or "something extract" for instance, I think it makes sense to keep the distinction.

Reply:
subtle, but ok